### PR TITLE
importing specific functions from SimpleGraphs to fix name clashes

### PR DESCRIPTION
**Pull request by millerjoey**
Testing caused some name clashes between `Distributions` and `SimpleGraphs` (#120). I imported the specific functions used from `SimpleGraphs` in core/toposort. They aren't used in core/model or core/utils so I removed that dependency.

Those are the only changes; git detects more in the diff but that's a false positive.

P.S. Let me know if I should merge into something other than master; I don't see dev anymore.

### CODE REVIEW

File: src/core/toposort.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 using SimplePosets
-using SimpleGraphs
+import SimpleGraphs: SimpleGraph, AbstractSimpleGraph, SimpleDigraph, vertex_type, NV, elist, in_neighbors, add_edges!
```
Comment (cscherrer) [inline src/core/toposort.jl:2]:
```suggestion
using SimpleGraphs: SimpleGraph, AbstractSimpleGraph, SimpleDigraph, vertex_type, NV, elist, in_neighbors, add_edges!
```

**Comment by millerjoey** [comment]
Ah right, didn't realize that difference. Yeah, `using` seems better in this case.

**Comment by cscherrer** [comment]
Sounds good, I'll merge once tests pass

**Comment by cscherrer** [comment]
Looks good to go, thanks @millerjoey !

**Pull request merged**